test/system/src/main/java/io/pravega/test/system/framework/services/PravegaSegmentStoreService.java

```diff
@@ -132,20 +136,34 @@ private App createPravegaSegmentStoreApp() {
         map.put("HDFS_URL", "hdfs.marathon.containerip.dcos.thisdcos.directory:8020");
         map.put("CONTROLLER_URL", conUri.toString());
         map.put("TIER2_STORAGE", "HDFS");
+        getCustomEnvVars(map, SEGMENTSTORE_EXTRA_ENV);
 
         //Properties set to override defaults for system tests
         String hostSystemProperties = setSystemProperty("autoScale.muteInSeconds", "120") +
                 setSystemProperty("autoScale.cooldownInSeconds", "120") +
                 setSystemProperty("autoScale.cacheExpiryInSeconds", "120") +
                 setSystemProperty("autoScale.cacheCleanUpInSeconds", "120") +
                 setSystemProperty("log.level", "DEBUG") +
-                setSystemProperty("curator-default-session-timeout", String.valueOf(30 * 1000)) +
-                setSystemProperty("hdfs.replaceDataNodesOnFailure", "false");
```

Comment:
This is very important when running with HDFS. Where are we setting it now?

Reply:
This was removed by mistake. Re-adding it.